src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/IssueFinalDecisionAboutToSubmitHandler.java

```diff
@@ -89,10 +89,14 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
             return preSubmitCallbackResponse;
         }
 
-        sscsCaseData.setDwpState(FINAL_DECISION_ISSUED.getId());
         createFinalDecisionNoticeFromPreviewDraft(preSubmitCallbackResponse);
         clearTransientFields(preSubmitCallbackResponse);
 
+        if (!(State.READY_TO_LIST.equals(sscsCaseData.getState())
+            || State.WITH_DWP.equals(sscsCaseData.getState()))) {
+            sscsCaseData.setDwpState(FINAL_DECISION_ISSUED.getId());
```

Comment:
is that mean other than Ready_to_list will be pass ?

Reply:
Any state except "Ready to list" or "with DWP"

Reply:
ok